Formula/apollo.rb

```diff
@@ -72,4 +74,10 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    system bin/"apollo", "create", testpath
+    assert_predicate testpath/"bin/apollo-broker", :exist?
```

Comment:
I started changing some of them before I left, but never followed through on it that consistently, so thought I'd try and adhere to that a little more 😉